src/Uno.UI/UI/Xaml/UIElement.macOS.cs

```diff
@@ -41,9 +43,25 @@ public UIElement()
 			InitializePointers();
 		}
 
-
+		public override void UpdateTrackingAreas()
```

Comment:
What's the purpose of this ? Would be great to have a small comment to explain this :)

Reply:
Will add it :-) . It is required to get the entered/exited/moved events to fire.

Reply:
Commented :-)